components/server/src/container-module.ts

```diff
@@ -264,19 +258,14 @@ export const productionContainerModule = new ContainerModule((bind, unbind, isBo
 
     bind(NewsletterSubscriptionController).toSelf().inSingletonScope();
 
-    bind(UsageServiceClientConfig).toDynamicValue((ctx) => {
+    bind<UsageServiceClient>(UsageServiceDefinition.name).toDynamicValue((ctx) => {
```

Comment:
@svenefftinge, shouldn't that be bound in singleton scope?

Asking, because of that one of the changes being introduced/deployed after which we see a severe memory leak.

Reply:
Yes, I think it can be in singleton scope as well. But if this increases leakage it could be there are other components that leak as well and hold references to these instances.